cmd/ddev/cmd/list_test.go

```diff
@@ -103,6 +103,12 @@ func TestCmdList(t *testing.T) {
 	siteList = getTestingSitesFromList(t, jsonOut)
 	assert.Equal(len(TestSites), len(siteList))
 
+	// Now filter the list by a not existing type
```

Comment:
Can we add one that uses existing type? I think there's always a wordpress type in TestCmd.

Reply:
I added a test but in general I have issues to run them on my local machine.